roles/config-rh-sso/defaults/main.yml

```diff
@@ -3,5 +3,6 @@
 rh_sso_host: "{{ ansible_facts.default_ipv4.address }}"
 rh_sso_port: 8080
```

Comment:
Would this value need to be updated when using https? ... or does it now listen on both http (8080) and https (8443)?

Reply:
Yes, thanks for pointing that out. The service listens to both http and https, however, it forces users to use https if SSL is enabled.